resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/DbReader.scala

```diff
@@ -160,7 +158,7 @@ object DbReader {
               NOT EXISTS (
                 SELECT *
                 FROM NODEPOOL np
-                LEFT JOIN APP a ON np.id = a.nodepoolId
+                RIGHT JOIN APP a ON np.id = a.nodepoolId
```

Comment:
This changes it to remove clusters with no nodepools and no apps.

Reply:
Yeah... its definitely confusing. I changed it back and forth and ran tests repeatedly